zipline/src/jniMain/kotlin/app/cash/zipline/QuickJs.kt

```diff
@@ -124,6 +130,47 @@ actual class QuickJs private constructor(
     return JniCallChannel(this, instance)
   }
 
+  actual fun collectModuleDependencies() {
```

Comment:
@swankjesse do we want these types of functions in QuickJs.kt or another file?

Reply:
Lets put them in another file, that way we get two benefits:
 - that file can be in an `internal` package, even if it isn‘t tagged with `internal` visibility
 - we can put it in engineMain and share it between platforms!